tests/data/power_flow/pgm-automatic-tap/params.json

```diff
@@ -0,0 +1,16 @@
+{
+  "calculation_method": "newton_raphson",
+  "tap_changing_strategy": "any_valid_tap",
+  "rtol": 1e-5,
+  "atol": 1e-5,
+  "extra_params": {
+    "newton_raphson": {
+      "experimental_features": "enabled",
+      "fail": {
+        "reason": "Experimental feature under development",
+        "raises": "PowerGridError",
+        "run": false
```

Comment:
why not just
```suggestion
        "raises": "AssertionError"
```
?

Reply:
It raises OSError which is not helpful at all. Maybe later, it will not raise OSError, but some other error.

Reply:
I see it fails with this in sonar cloud. Maybe skipping is the way to go
```
........................................................................ [ 22%]
Fatal Python error: Segmentation fault
```

Reply:
cfr #608 : not a problem in this PR. let's discuss whether we disable this PR or do it another way